### PR TITLE
staging gs manager url updates

### PR DESCRIPTION
**Pull request by pankaj9296**

### CODE REVIEW

File: config/local.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module.exports = {
-	BE_URL: 'http://localhost:8080'
+	BE_URL: 'http://localhost:8081'
 };
```
Comment (sloont) [inline config/local.js:3]: could you explain this change?

File: config/staging.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module.exports = {
-	BE_URL: 'https://staging.modd.io'
+	BE_URL: 'http://staging_gs_manager.modd.io'
 };
```
Comment (sloont) [inline config/staging.js:3]: why the change in name and to http?

File: config/local.js (hunk 1 above)
Comment (pankaj9296) [reply config/local.js:3]: Going forward, we will have GS manager as important part for local env setup.
So, all the communication from gs/taro will go to GS manager and not to the BE.
GS manager will run on http://localhost:8081
https://github.com/moddio/gs_manager
The setup steps are exactly the same as BE.

**Comment by sloont** [comment]
Awesome! Thanks for the clarification!

**Review by sloont — APPROVED** [review]
